server/lib/budget.js

```diff
@@ -221,37 +248,267 @@ export async function getTotalAmountPaidExpenses(collective, { startDate, endDat
+        'CollectiveId',
+      ]
+    : sequelize.col('CollectiveId');
+
+  const attributes = [
+    CollectiveId,
+    [sequelize.fn('COUNT', sequelize.col('Transaction.id')), 'count'],
+    [
+      sequelize.fn('COUNT', sequelize.fn('DISTINCT', sequelize.col('Transaction.FromCollectiveId'))),
+      'countDistinctFromCollective',
+    ],
+  ];
+
+  const group = [CollectiveId];
+
+  const results = await models.Transaction.findAll({
+    attributes,
+    where,
+    include,
+    group,
+    raw: true,
   });
 
-  const creditTotal = await sumTransactionsInCurrency(creditResults, currency);
+  for (const result of results) {
+    const CollectiveId = result['CollectiveId'];
+    const count = result['count'];
+    const countDistinctFromCollective = result['countDistinctFromCollective'];
+
+    // Initialize Collective total
+    if (!totals[CollectiveId]) {
+      totals[CollectiveId] = { CollectiveId, count: 0, countDistinctFromCollective: 0 };
+    }
+
+    totals[CollectiveId].count += count;
+    totals[CollectiveId].countDistinctFromCollective += countDistinctFromCollective;
+  }
+
+  return totals;
+}
+
+export async function getTotalNetAmountReceivedTimeSeries(
+  collective,
+  { loaders, startDate, endDate, timeUnit, currency, version, includeChildren } = {},
+) {
+  version = version || collective.settings?.budget?.version || DEFAULT_BUDGET_VERSION;
+  currency = currency || collective.currency;
+  let result;
+
+  // Optimized version using loaders
+  if (loaders && version === DEFAULT_BUDGET_VERSION) {
+    const netAmountReceivedTimeSeriesLoader = loaders.Collective.netAmountReceivedTimeSeries.buildLoader({
+      includeChildren,
+      startDate,
+      endDate,
+      timeUnit,
+    });
+    result = await netAmountReceivedTimeSeriesLoader.load(collective.id);
+  } else {
+    const results = await getSumCollectivesNetAmountReceived([collective.id], {
+      withTimeSeries: true,
+      timeUnit,
+      startDate,
+      endDate,
+      version,
+      includeChildren,
+    });
+
+    result = results[collective.id];
+  }
+
+  const fxRate = await getFxRate(result.currency, currency);
+  const nodes = Object.values(result.timeSeries).map(node => ({
+    date: node.date.toISOString(),
+    amount: { value: Math.round(node.value * fxRate), currency },
+  }));
+
+  // TODO: include the total value in the response?
+  return {
+    dateFrom: startDate,
+    dateTo: endDate,
+    timeUnit,
+    nodes,
+  };
+}
 
-  const feesResults = await sumCollectivesTransactions(collectiveIds, {
+export function getSumCollectivesNetAmountReceived(
+  collectiveIds,
+  {
     startDate,
     endDate,
+    version = DEFAULT_BUDGET_VERSION,
+    includeChildren = false,
+    withTimeSeries = false,
+    timeUnit,
+  } = {},
+) {
+  return sumCollectivesTransactions(collectiveIds, {
+    transactionType: 'NET_CREDIT',
+    startDate,
+    endDate,
+    // NOTE: What is netAmount vs amount? with and without host fees?
```

Comment:
In the past netAmount was amount - platform fees - host fees and payment processor fees.

But now that we don't have platform fees and host fees are separated transactions, this is only amount -  payment processor fees.

This is why we also have to remove host fees ourselves.